airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/release/pipeline.py

```diff
@@ -0,0 +1,86 @@
+import sys
+import anyio
+import dagger
+from pipelines.dagger.actions.remote_storage import upload_to_gcs
+
+PLATFORMS_TO_BUILD = {
+    "airbyte_ci_debian": "python:3.10-slim",
+    "airbyte_ci_macos": "arm64v8/python:3.10-slim",
+}
+
+DIRECTORIES_TO_MOUNT = [".git", "airbyte-ci"]
+
+async def create_airbyte_ci_release(
+    dagger_client: dagger.Client,
+    ci_gcs_credentials_secret: dagger.Secret,
+    ci_artifact_bucket_name: str,
+    image: str,
+    output_filename: str
```

Comment:
Originally it would have just been `airbyte-ci-debian` or `airbyte-ci-macos`

however I updated it to be `airbyte-ci-amd64-1.2.3` and `airbyte-ci-arm64-1.2.3`